docs/writing-docs/mdx.md

```diff
@@ -254,36 +285,67 @@ import Changelog from '../CHANGELOG.md';
 
 -->
 
-## Linking to other stories and pages
+## Troubleshooting
 
-When writing MDX, you may want to provide links to other stories or documentation pages and sections. You can use the `path` query string.
+### The migration seems flaky and keeps failing
 
-Considering a story with ID `some--id`, this redirects to the **Docs** tab of the story:
+By default, running the [migration command](#automigration) will try and migrate all existing MDX files in your project according to the MDX 2 specification. However, this might not always be possible, and you might run into issues during the migration. To help you troubleshoot those issues, we've prepared some recommendations that might help you.
```

Comment:
I disagree. We should instead leave it here instead of the public notion doc acting as a migration guide, as this will affect not only new users but also existing users. As most folks will not see it, by adding it, we have the added bonus of discoverability via searching the documentation. Also, to which the CLI should also point here.